docusaurus.config.js

```diff
@@ -98,5 +98,22 @@ module.exports = {
         },
       },
     ],
+  ],
+  plugins: [
+    [
+      '@docusaurus/plugin-client-redirects',
+      {
+        redirects: [
+          {
+            from: '/installation/ha',
```

Comment:
ahh great, thanks. I couldn't figure out how to do this when I was reorganizing things.